### PR TITLE
VACMS-7619: Create field_va_health_connect_phone but keep it hidden

### LINKED ISSUE #7619: Create field_va_health_connect_phone but keep it hidden
See #7618 for background. 
## ACs

- [x] VAMC system content type has a new field
  - [x] `field_va_health_connect_phone`  "VA health connect phone number" 
  - [x] Phone number field type
  - [x] Single value
- [x] Form settings
  - [x] Phone number formatter (settings TBD?)
  - [x] Form placement HIDDEN
  - [x] Help text: "To request changes, [contact CMS help desk](http://linktohelpdesk).
```
To request changes, <a href="mailto:support@va-gov.atlassian.net?subject=Inaccurate map thumbnail&amp;body=My VA health connect phone number is incorrect.">Email the CMS Support Team</a>
```
- [x] Node view: hidden.

### CMS Team

Please leave only the team that will do this work selected. 

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [x] `⭐️ Product support`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by netboss**
- Created "field_va_health_connect_phone" and configured it to be disabled on both manage form display and manage display.

## Description

closes #7619 

## Testing done

Passes all tests.

**Results of field help desk link test:**

>  —-—-—-—
Reply above this line.
Thanks for getting in touch with the Drupal CMS help desk.
Request description:
My VA health connect phone number is incorrect. ***THIS IS A TEST, PLEASE DISREGARD***
Please reply (to ivan.latorre@civicactions.com) to confirm this is working correctly. Thank you!
We'll take a look at your message, and will respond soon.
View request · Turn off this request's notifications
This is shared with Iván Latorre.
Powered by Jira Service Management 

## Screenshots
Fields:
VACMS-7619_fields

Manage form display:
VACMS-7619_manage_form_display

Manage display:
VACMS-7619_manage_display

## QA steps

As user administrator
1. [x] Go to `https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov//admin/structure/types/manage/health_care_region_page/fields` to verify and confirm the new `field_va_health_connect_phone` is listed there
2. [x] Go to `https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov//admin/structure/types/manage/health_care_region_page/form-display` to verify and confirm the new `field_va_health_connect_phone` is listed there and disabled in manage form display
3. [x] Go to `https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov//admin/structure/types/manage/health_care_region_page/display` to verify and confirm the new `field_va_health_connect_phone` is listed there and disabled in manage display
4. [x] Due to odd config changes... go to https://prod.cms.va.gov/central-texas-health-care and the tugboat version https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov/central-texas-health-care and do a side by side analysis to validate no changes.
5. [x] Due to odd config changes... go to https://prod.cms.va.gov/node/15023/edit and the tugboat version https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov/node/15023/edit and do a side by side analysis to validate no changes.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/behat/drupal-spec-tool/content_model_vamc_content_type_fields.feature [hunk 1]
```diff
@@ -238,4 +239,3 @@ Feature: Content model: VAMC Content Type fields
 | Content type | VAMC System Register for Care | VAMC System | field_office | Entity reference | Required | 1 | Select list | Translatable |
 | Content type | VAMC System Register for Care | Service | field_service_name_and_descripti | Entity reference |  | 1 | -- Disabled -- | Translatable |
 | Content type | VAMC System Register for Care | Non-clinical Services | field_non_clinical_services | Viewfield |  | 1 | Viewfield |  |
-
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_vamc_content_type_fields.feature:241]: Lost the trailing line

File: config/sync/core.entity_view_display.node.health_care_region_page.default.yml [hunk 2]
```diff
@@ -62,9 +62,7 @@ third_party_settings:
       label: Content
       region: content
     group_blurbs_for_subpages:
-      children:
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.health_care_region_page.default.yml:65]: There are a lot of changes appearing here that I would not expect.

File: config/sync/core.entity_view_display.node.health_care_region_page.default.yml [hunk 3]
```diff
@@ -249,8 +247,8 @@ content:
     region: content
 hidden:
   content_moderation_control: true
-  field_administration: true
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.health_care_region_page.default.yml:252]: This odd.  Nobody has touched this for months, but suddenly this gets removed as hidden.
image

Comment (swirtSJW) [reply config/sync/core.entity_view_display.node.health_care_region_page.default.yml:252]: It is indeed not hidden just seems odd that it has taken this long for config to catch up.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
I added some QA steps because:
1.  There weren't any that actually involved getting eyes on what the editors see.
2. There were a bunch of config changes that I did not expect to see that were possible config playing catch-up from changes long ago, but want to confirm with side-by-side comparison.

We want to get that trailing line back in the behat test.

File: config/sync/core.entity_view_display.node.health_care_region_page.default.yml (hunk 2 above)
Comment (netboss) [reply config/sync/core.entity_view_display.node.health_care_region_page.default.yml:65]: @swirtSJW 
I really don't know how/why these changes get there. The only thing I did in the manage display was moving the new `field_va_health_connect_phone` field to the disabled area. I didn't touch anything else.

I've looked at the CMS prod **Blurbs for subpages** group and compared it to my local and they both look the same. See screen shots.

CMS prod:
VAMC-System-fields-manage-display-prod

Local:
VAMC-System-fields-manage-display-local

And then I went to CMS prod config sync single export to get a copy of the actual config file to check and yes it's different:

CMS prod:
```
    group_blurbs_for_subpages:
      children:
        - field_featured_content_healthser
        - field_press_release_blurb
```

Local:
```
    group_blurbs_for_subpages:
      children: { }
```

I can't see a way to resolve this via the VAMC System content type manage display page. Can I just add the (above) code to the config manually?

File: tests/behat/drupal-spec-tool/content_model_vamc_content_type_fields.feature (hunk 1 above)
Comment (netboss) [reply tests/behat/drupal-spec-tool/content_model_vamc_content_type_fields.feature:241]: @swirtSJW 
In my local behat feature it have the trailing line. Not sure why this one doesn't have it. Should I try adding an extra (double) trailing line?

Comment (swirtSJW) [reply tests/behat/drupal-spec-tool/content_model_vamc_content_type_fields.feature:241]: It needs a single trailing line.  Your ide should be adding it on save.

Comment (netboss) [reply tests/behat/drupal-spec-tool/content_model_vamc_content_type_fields.feature]: @swirtSJW 
Updated it and re-pushed (in my local IDE I had to make it double trailing line).

**Comment by swirtSJW** [comment]
image

**Pull request merged**